src/app/static/src/app/store/baseline/baseline.ts

```diff
@@ -60,7 +60,33 @@ export const baselineGetters = {
         const validOrMissingPJNZ = !state.pjnzError
         const validOrMissingPop = !state.populationError
         return validOrMissingPJNZ && validOrMissingPop && state.validatedConsistent && !!state.shape
-    }
+    },
+    selectedDatasetAvailableResources: (state: BaselineState, getters: any, rooState: DataExplorationState) => {
+        const resources: { [k in keyof DatasetResourceSet]?: DatasetResource | null } = {}
+        const { selectedDataset } = state
+
+        if (selectedDataset?.id && selectedDataset.resources) {
+            const selectedDatasetFromDatasets = rooState.adr.datasets.find(dataset => dataset.id === selectedDataset.id) || null
+
+            const resourceTypes = {
+                pjnz: "inputs-unaids-spectrum-file",
+                pop: "inputs-unaids-population",
+                shape: "inputs-unaids-geographic",
+                survey: "inputs-unaids-survey",
+                program: "inputs-unaids-art",
+                anc: "inputs-unaids-anc"
+            }
+
+            const checkResourceAvailable = (resourceType: string) => {
+                return selectedDatasetFromDatasets?.resources.some((resource: any) => resource.resource_type && resource.resource_type === resourceType)
```

Comment:
I am unsure why you need `resource.resource_type && resource.resource_type === resourceType`, having to access resource.resource_type twice.

Reply:
I was being defensive, checking that resourceType existed on resource before comparing it.